mmv1/third_party/terraform/tests/resource_compute_network_edge_security_service_test.go.erb

```diff
@@ -0,0 +1,78 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccComputeNetworkEdgeSecurityService_update(t *testing.T) {
+	t.Parallel()
+
+	pName := fmt.Sprintf("tf-test-security-policy-%s", RandString(t, 10))
+	nesName := fmt.Sprintf("tf-test-edge-security-services-%s", RandString(t, 10))
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeNetworkEdgeSecurityServiceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkEdgeSecurityService_basic(pName, nesName),
+			},
+			{
+				ResourceName:      "google_compute_network_edge_security_service.foobar",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccNetworkEdgeSecurityService_update(pName, nesName),
+			},
+			{
+				ResourceName:      "google_compute_network_edge_security_service.foobar",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccNetworkEdgeSecurityService_basic(pName, nesName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_region_security_policy" "foobar" {
+  name        = "%s"
+  description = "basic region security policy"
+  type        = "CLOUD_ARMOR_NETWORK"
+  region      = "asia-southeast1"
```

Comment:
I managed to change most of them but I needed to left one behind because the tests run in parallel and this resource can have only one per region. I left this comment here to facilitate future maintainers.

Reply:
Sure! As I mentioned, it's not a blocker. We can leave it the way it is now if needed.